client/scss/components/_listing.scss

```
@@ -487,6 +487,66 @@ ul.listing {
     }
 }
 
+footer.bulk-actions-choices {
+    @include transition(bottom 0.1s ease 0.5s);
```

discussed in our catch up today:

```suggestion
    @include transition(bottom 0.1s ease 0.1s);
```

or plain `@include transition(bottom 0.1s);` for a quicker/smoother transition